### PR TITLE
feat(html): add grid spec 

### PR DESCRIPTION
**Pull request by epetrow**
closes: https://github.com/telerik/kendo-themes-private/issues/188

### CODE REVIEW

File: tests/grid/grid-locked-columns-rtl.html [hunk 1]
```diff
@@ -21,296 +21,305 @@
...
         <span>Angular Grid with locked columns, Grouping and Multiline headers</span>
-        <div dir="rtl" class="k-grid k-grid-md k-grid-lockedcolumns grid-styling-overrides" style="height: 700px;">
-            <div class="k-grid-aria-root">
-                <div class="k-grid-header">
-                    <div class="k-grid-header-locked" style="width: 582px;">
-                        <table class="k-table k-table-md k-grid-header-table" style="width: 582px;">
+        <div dir="rtl" class="k-grid-lockedcolumns grid-styling-overrides k-grid k-grid-md" style="height: 700px;">
+            <div class="k-grid-header">
+                <div class="k-grid-header-locked" style="width: 582px;">
+                    <table class="k-grid-header-table k-table k-table-md">
```
Comment (Juveniel) [inline tests/grid/grid-locked-columns-rtl.html:27]: inline style missing here

File: tests/grid/grid-column-reordering-actions.html [hunk 2]
```diff
@@ -176,46 +176,48 @@
...
-                            </tr>
-                            <tr class="k-master-row k-table-row">
-                                <td class="k-table-td">Perth Pasties</td>
-                                <td class="k-table-td">32.8</td>
-                                <td class="k-table-td">0</td>
-                                <td class="k-table-td">Meat/Poultry</td>
-                            </tr>
-                        </tbody>
-                    </table>
+                <div class="k-grid-container">
```
[the remark this answers is not in the document]
Comment (epetrow) [reply tests/grid/grid-column-reordering-actions.html:179]: The suites always render this container, so I decided to add it in the tests as well. It's present even if there is no locked content.

File: tests/_output/bootstrap/grid/grid-column-reordering.png
[the remark this answers is not in the document]
Comment (epetrow) [reply tests/_output/bootstrap/grid/grid-column-reordering.png:1]: I've also noticed this, however this does not seem to appear when I run the tests locally. Aside from correcting the rendering of the header cells there is no difference. I've tested this with the lastest version of Firefox as well and the change in the border is not present.

File: tests/grid/grid-adaptive-pager.html [hunk 3]
```diff
@@ -15,8 +15,8 @@
                 <div class="k-pane">
                     <div class="k-view k-widget">
                         <div class="k-stretched-view k-content">
-                            <div class="k-grid k-grid-md k-grid-display-block k-grid-mobile k-editable">
-                                <div class="k-grid-pager k-pager k-pager-md">
+                            <div class="k-grid-display-block k-grid-mobile k-editable k-grid k-grid-md">
+                                <div class="k-grid-pager k-grid-pager k-pager k-pager-md">
```
Comment (inikolova) [inline tests/grid/grid-adaptive-pager.html:19]: "_k-grid-pager_" class is duplicated

File: packages/html/src/grid/tests/grid-adaptive-pager.tsx [hunk 4]
```diff
@@ -11,9 +11,10 @@ export default () =>(
                     <div className="k-pane">
                         <div className="k-view k-widget">
                             <div className="k-stretched-view k-content">
-                                <div className="k-grid k-grid-md k-grid-display-block k-grid-mobile k-editable">
-                                    <Pager className="k-grid-pager" adaptive info={false} pageSizes={false} />
-                                </div>
+                                <Grid className="k-grid-display-block k-grid-mobile k-editable" pager={(
+                                    <GridPager className="k-grid-pager" adaptive info={false} pageSizes={false} />
```
Comment (inikolova) [inline packages/html/src/grid/tests/grid-adaptive-pager.tsx:15]: Remove className="k-grid-pager" attribute

File: packages/html/src/grid/tests/grid-adaptive-pager.tsx [hunk 5]
```diff
@@ -26,9 +27,10 @@ export default () =>(
                     <div className="k-pane">
                         <div className="k-view k-widget">
                             <div className="k-stretched-view k-content">
-                                <div className="k-grid k-grid-md k-grid-display-block k-grid-mobile k-editable">
-                                    <Pager className="k-grid-pager" adaptive info={false} pageSizes={false} dir="rtl" />
-                                </div>
+                                <Grid className="k-grid-display-block k-grid-mobile k-editable" pager={(
+                                    <GridPager className="k-grid-pager" adaptive info={false} pageSizes={false} dir="rtl" />
```
Comment (inikolova) [inline packages/html/src/grid/tests/grid-adaptive-pager.tsx:31]: Same as the above

File: packages/html/src/grid/tests/grid-editing-inline-angular.tsx [hunk 6]
```diff
@@ -1,168 +1,160 @@
...
-                                    </tr>
-                                </thead>
-                            </table>
+                                <TableTbody>
+                                    <TableRow className="k-grid-add-row k-grid-edit-row" role="row">
+                                        <TableTd colspan={1}></TableTd>
+                                        <TableTd colspan={1}>
+                                            <Textbox />
+                                        </TableTd>
+                                        <TableTd className="k-command-cell">
```
Comment (inikolova) [inline packages/html/src/grid/tests/grid-editing-inline-angular.tsx:120]: Is colSpan={1} missed on purpose?

File: packages/html/src/grid/tests/grid-filter-menu-angular.tsx [hunk 7]
```diff
@@ -1,204 +1,201 @@
...
+                                <col />
+                                <col />
+                            </colgroup>
+                            <TableThead>
+                                <TableRow>
+                                    <GridHeaderCell resizable rowspan={1} colspan={1}>
+                                        <Checkbox />
+                                    </GridHeaderCell>
+                                    <GridHeaderCell resizable rowspan={1} colspan={1} sortable menu="filter" columnTitle="Name"></GridHeaderCell>
+                                    <GridHeaderCell resizable rowspan={1} colspan={1} columnTitle="Command"></GridHeaderCell>
```
Comment (inikolova) [inline packages/html/src/grid/tests/grid-filter-menu-angular.tsx:152]: Is this cell resizable on purpose?

Comment (epetrow) [reply packages/html/src/grid/tests/grid-filter-menu-angular.tsx:152]: Yes, there was a mistake in the original test, the example was titled as resizable however there was no column-resizer, hence the resizable cell

**Review by inikolova — APPROVED** [review]

**Pull request merged**